chromium_src/chrome/browser/history/history_utils_unittest.cc

```diff
@@ -33,4 +33,6 @@ TEST(HistoryUtilsTest, VariousURLTest) {
   EXPECT_FALSE(CanAddURLToHistory(GURL("chrome-search://test")));
   EXPECT_FALSE(CanAddURLToHistory(GURL("chrome-distiller://test")));
   EXPECT_FALSE(CanAddURLToHistory(GetSampleLitePageUrl()));
+  EXPECT_FALSE(CanAddURLToHistory(
```

Comment:
Adding extension guard in here also would be good.

Reply:
done